packages/design-system/src/ui/components/button/button.tsx

```diff
@@ -1,8 +1,8 @@
+import { cn } from "../../../utils/cn.js";
```

Comment:
there is something wrong with this component but I can't figure out what it is. `import { cn } from "@powerhousedao/design-system";` works everywhere else, but in this component doing that causes tests to fail with "cn is not a function".